lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss

```diff
@@ -157,4 +160,23 @@
     }
   }
 
-}
\ No newline at end of file
+}
+
+
+
+// @TODO: move to proper locations
+// Temporary additions to work through fix for upload button issues.
+input[type="file"] {
+  @include visually-hidden();
+}
+
+.button--file {
+  height: auto;
+  width: auto;
+}
+
+.gigantor {
+  background-color: #f0f;
```

Comment:
The gigantor stuff is still in the works. I just wanted to get this PR in, cause it'll likely affect the Image Crop stuff Shae's working on, before I continue with the gigantor styles.